src/lib/components/Games/HolidayGame.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { browser } from '$app/environment';
+	import { holidayLists } from '$lib/assets/holidayLists';
+	import Board from '$lib/components/Board.svelte';
+	import GameSummary from '$lib/components/GameSummary.svelte';
+	import Timer from '$lib/components/Timer.svelte';
+	import { gameProgress } from '$lib/store';
+	import { formWord, makeWordUsed, type LetterData } from '$lib/utils/letterDataUtils';
+	import { createLetterData } from '$lib/utils/letterDataUtils';
+	import { checkIfValidWord } from '$lib/utils/wordUtils';
+
+	let now = new Date();
+	const wordList = holidayLists.find((list) => {
+		return now.getMonth() == list.date.getMonth() && now.getDate() == list.date.getDate();
+	})?.words;
+
+	if (wordList == undefined) {
+		throw new Error("No holiday found for today's date");
+	}
```

Comment:
should this really throw? does this somehow fall back to the default game? or does the app just not work if this happens?

Reply:
Yeah I am 50/50 on this. This will only throw if somehow the homepage promises a specific holiday game, you load that holiday, and the data is somehow lost in between that time. To me that feels like an actual error, but maybe it is just better practice to gracefully redirect to the normal mode.

Reply:
I'd vote for graceful redirect

Reply:
Now added